client/app/bundles/course/survey/components/QuestionForm.jsx

```diff
@@ -0,0 +1,296 @@
+    const { disabled } = this.props;
+    return (
+      <Field
+        component={TextField}
+        type="number"
+        style={styles.numberOfResponsesField}
+        inputStyle={styles.numberOfResponsesInput}
+        underlineStyle={styles.numberOfResponsesInput}
+        {...{ name, floatingLabelText, disabled }}
+      />
+    );
+  }
+
+  renderSpecificFields(questionType) {
+    const { intl, disabled } = this.props;
+    const { TEXT, MULTIPLE_CHOICE, MULTIPLE_RESPONSE } = questionTypes;
+
+    let specificFields;
+    switch (questionType) {
+      case MULTIPLE_CHOICE:
+        specificFields = (
+          <FieldArray name="options" component={this.renderMcqOptions} {...{ disabled }} />
+        );
+        break;
+      case MULTIPLE_RESPONSE:
+        specificFields = (
+          <div>
+            <div style={styles.numberOfResponsesDiv}>
+              {this.renderNumberOfResponsesField(
+                'min_options',
+                intl.formatMessage(translations.minOptions)
+              )}
+              {this.renderNumberOfResponsesField(
+                'max_options',
+                intl.formatMessage(translations.maxOptions)
+              )}
+            </div>
+            <FieldArray name="options" component={this.renderMrqOptions} {...{ disabled }} />
+          </div>
+        );
+        break;
+      case TEXT:
+      default:
+        specificFields = [];
+    }
+    return specificFields;
+  }
+
+  render() {
+    const { handleSubmit, intl, onSubmit, disabled, formValues } = this.props;
+    const { TEXT, MULTIPLE_CHOICE, MULTIPLE_RESPONSE } = questionTypes;
+    const questionType = formValues && formValues.question_type;
+
+    return (
+      <Form onSubmit={handleSubmit(onSubmit)}>
+        <div style={styles.fieldRow}>
+          <Field
+            name="question_type"
+            floatingLabelText={intl.formatMessage(translations.questionType)}
+            component={SelectField}
+            style={styles.questionType}
+            {...{ disabled }}
+          >
+            <MenuItem
+              value={TEXT}
+              primaryText={intl.formatMessage(translations.textResponse)}
+            />
+            <MenuItem
+              value={MULTIPLE_CHOICE}
+              primaryText={intl.formatMessage(translations.multipleChoice)}
+            />
+            <MenuItem
+              value={MULTIPLE_RESPONSE}
+              primaryText={intl.formatMessage(translations.multipleResponse)}
+            />
+          </Field>
+          <Field
+            name="required"
+            label={intl.formatMessage(questionFormTranslations.compulsory)}
+            component={Toggle}
+            style={styles.requiredToggle}
+            labelStyle={styles.requiredToggleLabel}
+            {...{ disabled }}
+          />
+        </div>
+        <Field
+          name="description"
+          floatingLabelText={intl.formatMessage(translations.question)}
```

Comment:
<img width="710" alt="screen shot 2017-02-02 at 7 52 23 pm" src="https://cloud.githubusercontent.com/assets/6252919/22548404/2dff8e1e-e981-11e6-9163-014c56287194.png">


I used "Question" because I thought it was clearer. If we use "Description", it can seem that we are asking the user to describe the question and it is technically correct to say that a question is "good" or "outrageous" etc. The user has to figure out that is not what is intended by looking at the context. If we use "Question" or perhaps "Question Text", the user does not have to make the inference.

If this is okay, we can change the database column also.